external/src/main/java/com/redhat/cloud/policies/engine/process/Receiver.java

```diff
@@ -59,19 +60,34 @@
     @Metric(absolute = true, name = "engine.input.processed.errors", tags = {"queue=host-egress"})
     Counter processingErrors;
 
-    @Incoming("host-egress")
+    @Incoming("events")
     @Acknowledgment(Acknowledgment.Strategy.MANUAL)
     public CompletionStage<Void> processAsync(Message<String> input) {
         return
                 CompletableFuture.supplyAsync(() -> {
                     // smallrye-messaging 1.1.0 and up has its own metric for received messages
                     incomingMessagesCount.inc();
-                    if(log.isTraceEnabled()) {
+                    if (log.isTraceEnabled()) {
                         log.tracef("Received message, input payload: %s", input.getPayload());
                     }
                     JsonObject json = new JsonObject(input.getPayload());
                     return json;
                 }).thenApplyAsync(json -> {
+                    if (json.containsKey(TYPE_FIELD)) {
+                        String eventType = json.getString(TYPE_FIELD);
+                        if (eventType.equals("delete")) {
```

Comment:
Should we do a !eventType.equals("insert") here ? (I don't know the actual name of the event) In case they add more types ?

Reply:
Wouldn't we miss the 1st report for a new system then?

Reply:
I guess we need to listen to "created" and "updated" and ignore the rest (only deleted now) ?
Maybe I'm just extra cautious, I'm afraid a new type gets added and we choke on it

Reply:
Ok.